src/gui/PlaylistDock.cpp

```diff
@@ -285,30 +297,32 @@ void PlaylistDock::next(bool playingError)
 					for (;;)
 					{
 						tWI = list->itemBelow(tWI);
-						if (canRepeat && repeatMode == RepeatGroup && P && (!tWI || tWI->parent() != P)) //zapętlenie grupy
+						if (canRepeat && repeatMode == RepeatGroup && P && (!tWI || tWI->parent() != P)) //loop group
 						{
 							const QList<QTreeWidgetItem *> l2 = list->getChildren(PlaylistWidget::ONLY_NON_GROUPS, P);
 							if (!l2.isEmpty())
 								tWI = l2[0];
 							break;
 						}
-						if (PlaylistWidget::isGroup(tWI)) //grupa
+						if (PlaylistWidget::isGroup(tWI)) //group
 						{
 							if (!tWI->isExpanded())
 								tWI->setExpanded(true);
 							continue;
 						}
+						if (tWI->font(0).strikeOut()) //skip item
```

Comment:
Is it a good idea to check font attributes? Maybe a flag in user data or something?